.travis.yml

```diff
@@ -20,10 +18,14 @@ php:
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - php: hhvm
```

Comment:
Hi @stof, thank you for suggestions again.
Now the HHVM is fixed and remove the ```allow_failures```.
Thanks.